freqtrade/freqtradebot.py

```diff
@@ -292,6 +292,8 @@ def _get_min_pair_stake_amount(self, pair: str, price: float) -> Optional[float]
 
         # reserve some percent defined in config (5% default) + stoploss
         amount_reserve_percent = 1.0 - self.config.get('amount_reserve_percent', 0.05)
+        amount_reserve_percent = 1.0 - self.config.get('amount_reserve_percent',
```

Comment:
Whilest i agree, you should not duplicate the line, but replace the line instead.

I'm quite certain it was the intent anyway - happens to me too all the time ;)

Reply:
Oops, it'a sleep, sure.
I made same changes 3, 4 or 5 times in my fork, removing the unsuccessful working branches, since result of git rebase was different from what I want to achieve (only 1 commit with these changes for PR).
So on 4th time I've simply forgotten to remove the old line...